jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskdetailsmulti/TaskDetailsMultiPresenter.java

```diff
@@ -151,6 +162,18 @@ public void onTaskSelectionEvent(@Observes final TaskSelectionEvent event) {
         containerId = event.getContainerId();
         processId = event.getTaskName();
 
+        breadcrumbs.createListDetailBreadcrumbs(placeManager,
```

Comment:
so creating both admin and regular task list, I'm wondering if we should consider breaking these screens into two?